diagnostic.md

```diff
@@ -27,7 +27,7 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+$body-base-font-size: h1(base-font-siz, 30%);
```

Comment:
Sorry, that's not correct.
```
h1 {
  font-size: ....
}
```
sets the font size of h1 elements.
Specifically we want it to set it to 30% of `$base-font-size`, so we'd write:
```
h1 {
  font-size: $base-font-size * 1.3;
```